docs/triggers/shortcut-triggers.md

```diff
@@ -0,0 +1,125 @@
+## Shortcut Triggers
+
+A shortcut trigger is a trigger that activates when a shortcut is clicked in the Slack client. When a shortcut trigger is created, its API response returns a `shortcut_url` which can be pased into the Slack client causing the client to render it as a button. Clicking on this button will activate the associated workflow. A shortcut trigger
+includes the common [trigger attributes](./trigger-basics.md#trigger-types) along with an optional shortcut parameter: 
+
+| Parameter name  | Required?     | Description                                                          |
+| ----------------|:-------------:| ---------------------------------------------------------------------|
+| `shortcut`        | No            | Contains information about the [button text](#shortcut-object) of the shortcut          |
+
+### Shortcut Object
+
+A Shortcut trigger can contain an optional shortcut object which specifies additional details about the shortcut. Currently, the shortcut object is used to specify the button text of the shortcut associated with the trigger and has the following shape:
+
+```ts
+  shortcut?: {
+    button_text: string;
+  };
+```
+
+### Shortcut Data Context Object
+The Shortcut trigger also has access to a data context object which includes information related to the context of the trigger activation. This data object can be used to fill the optional input fields of the trigger being activated. The data context parameters available to a shortcut trigger are as follows:
```

Comment:
Lets link to the Data Context Object section from `trigger-basics`.  Then we can lean on the assumption they're familiar with it and just provide details.

```suggestion
The `data` context parameters available to a Shortcut Trigger are as follows:
```